### ISSUE TITLE
installing on Raspberry Pi 

### ISSUE THREAD

**Issue by robynjayqueerie** [Cloud-Automation/node-modbus#196]
I am trying to use this on Pi 3B with Stretch. Have tried all sorts of install using npm from npm install jsmodbus locally through global install and sudo etc. But I can't get my let modbus = require('jsmodbus') to find the package. Have tried using paths in that statement to the node_modules repository for jsmodbus but still no luck.
The installation suggests that it has installed jsmodbus@3.0.0 each time and the jsmodbus directory turns up in the appropriate node_modules directory.
Basically node throws an error saying it can't find jsmodbus. Do you have any ideas, I am running the latest node.js (ver 10.2.1) and npm (ver 6.1.0)
Thanks
Robyn

**Comment by robynjayqueerie** [comment]
After several attempts at installing it locally it worked, I really don't know what changed. So I guess we close this now unless someone else has had some experience and can throw light on the issue.
Thanks
